src/reducers/reducer.js

```diff
@@ -1229,6 +1229,7 @@ export const onReloadWalletRequested = (state) => ({
   tokensHistory: initialState.tokensHistory,
   tokensBalance: initialState.tokensBalance,
   loadHistoryStatus: initialState.loadHistoryStatus,
+  nanoContract: initialState.nanoContract,
```

Comment:
I think we should improve this docstring. I thought this reload was for any wallet reload, e.g. when the wallet loses internet connection. But this is only after you connect to a new network settings, right? I think this should be explicit in the docstring